modules/nw-metallb-community-cr.adoc

```diff
@@ -0,0 +1,57 @@
+
+:_content-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-configure-address-pools.adoc
+[id="nw-metallb-community-cr_{context}"]
+= About the community custom resource
+
+The community custom resource is a collection of aliases for communities. Users can define named aliases to be used when advertising `ipAddressPools` using the `BGPAdvertisement`. The fields for the community custom resource are described in the following table.
```

Comment:
In the body text (but not the heading where we omit markdown), I think `community` should be in back-ticks per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references